engine-agent/src/main/java/io/camunda/zeebe/process/test/engine/agent/AgentProperties.java

```diff
@@ -0,0 +1,41 @@
+package io.camunda.zeebe.process.test.engine.agent;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AgentProperties {
+
+  private static final Logger LOG = LoggerFactory.getLogger(AgentProperties.class);
+
+  private static final String PROPERTIES_FILE = "/config.properties";
+  public static final String GATEWAY_PORT = "gateway.port";
+  public static final String CONTROLLER_PORT = "controller.port";
+  private static final Properties PROPERTIES = new Properties();
+
+  static {
+    try (final InputStream inputStream = AgentProperties.class.getResourceAsStream(PROPERTIES_FILE)) {
+      PROPERTIES.load(inputStream);
+    } catch (FileNotFoundException e) {
+      LOG.error("Could not find property file with name " + PROPERTIES_FILE, e);
+      throw new RuntimeException(e);
```

Comment:
🔧 It seems that from now on the config file is required to use this project. Please make it clear in the error message what is expected from the user to fix the problem.